common/repo/GitRepo.java

```diff
@@ -381,6 +381,15 @@ public String getHeadCommitId() {
     return commandResult.stdout.trim().replace("\n", "");
   }
 
+  // Retrieves the URLs for a remote (e.g. https://github.com/google/startup-os.git)
+  public String getRemoteURL() {
+    return runCommand("remote get-url origin").stdout;
+  }
+
+  public boolean hasChanges() {
+    return !getUncommittedFiles().isEmpty();
```

Comment:
There is an unexpected behavior of `getCommits()` method. 
I've described the issue in comments. Please look at it.

Reply:
I looked, so `getCommits()` and you're right. Could you please change the documentation in `Repo.java` from:
`Gets commits on branch since it diverged from master`
To:
`Gets commits on branch since it diverged from master, including the last master branch, at position 0`

In `hasChages()`, I suggest you ignore the first element and document why you do.

Reply:
Done.
Did you mean "Gets commits on branch since it diverged from master, including the last master ~~branch~~ commit, at position 0", right?